src/main/kotlin/christmas/ui/Input.kt

```diff
@@ -0,0 +1,48 @@
+package christmas.ui
+
+object Input {
+
+    fun enterReservationDate(): Int {
+        println(
+            """
+            안녕하세요! 우테코 식당 12월 이벤트 플래너입니다.
+            12월 중 식당 예상 방문 날짜는 언제인가요? (숫자만 입력해 주세요!)
+        """.trimIndent()
+        )
+        try {
+            val day = readLine()!!.toInt()
+            if (day < 0 || day > 31) {
+                throw IllegalArgumentException("존재하지 않는 날짜입니다.")
+            }
+            return day
+        } catch (e: NumberFormatException) {
+            throw IllegalArgumentException("숫자 형식이 아닙니다.")
+        }
+    }
+
+    fun enterMenus(): List<String> {
+        println("주문하실 메뉴를 메뉴와 개수를 알려 주세요. (e.g. 해산물파스타-2,레드와인-1,초코케이크-1)")
+        val input = readLine().orEmpty()
+        return parseMenuInput(input)
+    }
+
+    private fun parseMenuInput(input: String): List<String> {
+        if (input.isBlank()) return emptyList()
+
+        val menuTokens = input.split(",")
+
+        return menuTokens.flatMap { token ->
+            parseMenuToken(token)
```

Comment:
```suggestion
        return menuTokens.flatMap(::parseMenuToken)
```
Could you explain the reason for using flatMap?

Reply:
If I had used a for loop, the parsing logic might have been more straightforward.
But since I'm still in the learning phase, I wanted to try using functional approaches as much as possible. 😅